plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiSmokeTest.java

```diff
@@ -65,6 +68,13 @@ public void testReadPartitionedTables()
                 "SELECT * FROM VALUES ('2018-08-31', '99')");
     }
 
+    @Test
+    public void testIfExceptionIsThrownProperly()
+    {
+        assertQueryFailsEventually("SELECT dt, count(1) FROM " + STOCK_TICKS_MOR_EXCEPTION_TEST + " GROUP BY dt",
```

Comment:
Why "eventually"?

Reply:
because query wont hang and it will fail eventually @findepi

Reply:
why not `assertQueryFails` then?

Reply:
did not tried with assertQueryFails I will try with it

Reply:
@findepi  done